controllers/space/space_controller_test.go

```diff
@@ -1270,49 +1425,70 @@ func requestFor(s *toolchainv1alpha1.Space) reconcile.Request {
 	}
 }
 
-func TestNewNsTemplateSetSpec(t *testing.T) {
-	t.Run("when clusterResources template is specified", func(t *testing.T) {
-		// given
-		nsTemplateTier := tiertest.NewNSTemplateTier("advanced", "dev", "stage", "extra")
-
-		// when
-		setSpec := space.NewNSTemplateSetSpec(nsTemplateTier)
-
-		// then
-		assert.Equal(t, newExpectedNsTemplateSetSpec(), setSpec)
-	})
+func TestNewNSTemplateSetSpec(t *testing.T) {
 
-	t.Run("when clusterResources template is NOT specified", func(t *testing.T) {
+	t.Run("full monty", func(t *testing.T) {
```

Comment:
its not clear to me what is full monty, could you please clarify or consider changing the name of this test case?

Reply:
right, I did not realize that when I started working on this test. I removed the subtest wrapper in ba329fa, since there was no need for it anyways.